homeassistant/components/light/lw12wifi.py

```diff
@@ -0,0 +1,173 @@
+"""
+Support for Lagute LW-12 WiFi LED Controller.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.lw12wifi/
+"""
+import logging
+import time
+
+import voluptuous as vol
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, ATTR_EFFECT, ATTR_HS_COLOR, ATTR_TRANSITION,
+    PLATFORM_SCHEMA, SUPPORT_BRIGHTNESS, SUPPORT_COLOR, SUPPORT_EFFECT,
+    SUPPORT_TRANSITION, Light)
+from homeassistant.const import (
+    CONF_BRIGHTNESS, CONF_EFFECT, CONF_HOST, CONF_NAME, CONF_PORT, CONF_RGB)
+import homeassistant.helpers.config_validation as cv
+import homeassistant.util.color as color_util
+
+REQUIREMENTS = ['lw12==0.9.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_TRANSITION = 'transition'
+
+DEFAULT_BRIGHTNESS = 255
+DEFAULT_EFFECT = None
+DEFAULT_NAME = 'LW-12 FC'
+DEFAULT_PORT = 5000
+DEFAULT_RGB = [255, 255, 255]
+DEFAULT_TRANSITION = 128
+
+SUPPORT_LW12 = (
+    SUPPORT_BRIGHTNESS | SUPPORT_EFFECT | SUPPORT_COLOR |
+    SUPPORT_TRANSITION
+)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_BRIGHTNESS, default=DEFAULT_BRIGHTNESS):
```

Comment:
Should be removed.

Reply:
If you want to restore the previous known state after a restart, use `helpers/restore_state.py` (but do that in a new PR)